lib/player_widget.dart

```diff
@@ -255,13 +255,9 @@ class _PlayerWidgetState extends State<PlayerWidget> {
     return result;
   }
 
-
   Future<int> _rewind() async {
     Duration _tempPosition = Duration(seconds: _position!.inSeconds - 10);
```

Comment:
max(0, _position!.inSeconds - 10)

Reply:
that's one of my edge cases I was working on next :) added min to forward as well.